OpenRA.Mods.Common/Traits/Buildings/Reservable.cs

```diff
@@ -71,17 +78,27 @@ public static bool IsAvailableFor(Actor reservable, Actor forActor)
 			return res == null || res.reservedForAircraft == null || res.reservedForAircraft.MayYieldReservation || res.reservedFor == forActor;
 		}
 
-		private void UnReserve()
+		void UnReserve(Actor self)
 		{
 			if (reservedForAircraft != null)
-				reservedForAircraft.UnReserve(true);
+			{
+				if (reservedForAircraft.GetActorBelow() == self)
+				{
+					if (rallyPoint != null)
+						reservedFor.QueueActivity(reservedForAircraft.MoveTo(rallyPoint.Location, null, targetLineColor: Color.Green));
```

Comment:
Aircraft  (and other units) who immediately leave the host after resupplying already do a regular move instead of an attack-move, so I opted for consistency with resupplying instead of unit production.

Reply:
> Aircraft (and other units) who immediately leave the host after resupplying already do a regular move instead of an attack-move

We could change it so that they always leave with an attack move. But this discussion does not need to block this PR IMO.

Reply:
? I just checked bleed and a helicopter moving to the rally point after rearming does attack move.

Reply:
That is actually because of the same bug as in #17054. It is defined in `Resupply` as a regular move, but that is never properly executed on bleed because of the bogus attack-move in `TakeOff`.